src/kaldi_recognizer.cc

```diff
@@ -423,6 +424,7 @@ const char *KaldiRecognizer::MbrResult(CompactLattice &clat)
         word["start"] = samples_round_start_ / sample_frequency_ + (frame_offset_ + times[i].first) * 0.03;
         word["end"] = samples_round_start_ / sample_frequency_ + (frame_offset_ + times[i].second) * 0.03;
         word["conf"] = conf[i];
+        confidence += conf[i];
```

Comment:
I don't believe it should work this way. Assuming a confidence is a value in [0;1] range, you can't just sum all the values to get an overall confidence.

Reply:
Fixed the overall confidence.